app/src/WebsiteGenerator/ContentTypeGenerator/StaticPagesGenerator.php

```diff
@@ -54,7 +54,7 @@ public function generatePages(TemplateRenderer $templateRenderer)
 
     private function generateStaticPage(TemplateRenderer $templateRenderer, Route $route, string $template)
     {
-        $filename = $route->getPath().'.html';
+        $filename = $route->getPath().'/index.html';
```

Comment:
This does not appear to be necessary. At least it isn't when running a VM provisioned using the ansible scripts.